icons/pfsense/pfsense-original-wordmark.svg

```diff
@@ -0,0 +1 @@

```

Comment:
Add ` fill="#1a1a1a"`
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#1a1a1a" d="M3.059 45.765C1.365 45.765 0 47.176 0 48.87v27.012L4.941 58.33H9.93l-.659 2.306h.095c.282-.33.658-.612 1.129-.941.47-.33.941-.612 1.46-.894.517-.283 1.081-.47 1.693-.658.612-.189 1.224-.236 1.836-.236 1.317 0 2.4.235 3.34.658.942.424 1.647 1.083 2.212 1.883.47.706.753 1.553.894 2.541l.007-.005c.009.092.04.205.04.287.094 1.083-.047 2.307-.424 3.624-.33 1.224-.846 2.4-1.552 3.53a15.02 15.02 0 0 1-2.4 2.964 12.346 12.346 0 0 1-3.106 2.07c-1.13.518-2.306.753-3.577.753a8.594 8.594 0 0 1-3.012-.517c-.894-.33-1.506-.94-1.882-1.788H5.93l-2.353 8.33h29.789a3.103 3.103 0 0 0 3.106-3.106V52.917c-.142-.047-.283-.093-.377-.14a4.211 4.211 0 0 0-.847-.095c-.847 0-1.553.189-2.117.565-.518.377-.989 1.13-1.318 2.306l-.8 2.776h3.059l.753 2.4-1.977 1.93h-3.06l-3.716 13.176h-5.365l3.718-13.176h-2.071l-.07.047h-.072v-.047l1.224-4.33H25.6l.847-3.106c.282-.94.612-1.881 1.036-2.729.376-.847.94-1.6 1.599-2.259.659-.659 1.506-1.177 2.494-1.553.989-.376 2.26-.564 3.718-.564.377 0 .705 0 1.082.047a3.163 3.163 0 0 0-2.964-2.4zm43.718 12.093c-.894 0-1.836.095-2.683.33-.894.236-1.694.565-2.353 1.082-.706.47-1.224 1.083-1.647 1.836-.424.753-.611 1.647-.611 2.683 0 1.082.234 1.976.658 2.635a4.489 4.489 0 0 0 1.694 1.552c.706.377 1.412.612 2.212.8.8.189 1.505.33 2.211.518.706.141 1.272.33 1.695.564.424.236.658.566.658 1.036 0 .47-.235.8-.753.988a4.921 4.921 0 0 1-1.693.283c-.8 0-1.553-.142-2.165-.471-.612-.33-1.176-.8-1.694-1.412l-3.294 3.624c.847.894 1.929 1.505 3.153 1.835 1.27.33 2.494.518 3.717.518.942 0 1.883-.095 2.87-.283.942-.188 1.836-.47 2.589-.941a5.265 5.265 0 0 0 1.882-1.788c.47-.753.753-1.647.753-2.776 0-1.083-.235-1.977-.659-2.636-.47-.611-1.035-1.176-1.694-1.553a8.962 8.962 0 0 0-2.211-.846c-.8-.189-1.507-.33-2.213-.519a8.13 8.13 0 0 1-1.693-.517c-.424-.188-.66-.518-.66-.941 0-.518.236-.895.66-1.13A2.735 2.735 0 0 1 46.824 62c1.317 0 2.352.47 3.152 1.459l3.294-3.294c-.8-.848-1.835-1.459-3.012-1.788-1.176-.33-2.352-.519-3.481-.519zm54.775 0c-.894 0-1.835.095-2.682.33-.894.236-1.694.565-2.353 1.082-.705.47-1.223 1.083-1.647 1.836-.423.753-.612 1.647-.612 2.683 0 1.082.236 1.976.66 2.635a4.486 4.486 0 0 0 1.693 1.552c.706.377 1.413.612 2.213.8.8.189 1.505.33 2.21.518.707.141 1.271.33 1.695.564.423.236.659.566.659 1.036 0 .47-.235.8-.753.988a4.921 4.921 0 0 1-1.694.283c-.8 0-1.553-.142-2.164-.471-.612-.33-1.178-.8-1.695-1.412l-3.294 3.624c.847.894 1.93 1.505 3.153 1.835 1.27.33 2.494.518 3.718.518.941 0 1.882-.095 2.87-.283.942-.188 1.836-.47 2.589-.941A5.265 5.265 0 0 0 108 73.247c.47-.753.753-1.647.753-2.776 0-1.083-.236-1.977-.66-2.636-.47-.611-1.034-1.176-1.693-1.553a8.963 8.963 0 0 0-2.212-.846c-.8-.189-1.506-.33-2.212-.519a8.13 8.13 0 0 1-1.694-.517c-.423-.188-.659-.518-.659-.941 0-.518.236-.895.66-1.13A2.735 2.735 0 0 1 101.6 62c1.318 0 2.353.47 3.153 1.459l3.294-3.294c-.8-.848-1.836-1.459-3.012-1.788-1.177-.33-2.353-.519-3.483-.519zm-36.517.048c-1.365 0-2.635.235-3.812.658a9.36 9.36 0 0 0-3.105 1.836c-.894.8-1.553 1.789-2.071 2.918-.518 1.13-.753 2.4-.753 3.765 0 1.411.235 2.635.753 3.764.518 1.13 1.177 2.118 2.07 2.918a9.36 9.36 0 0 0 3.106 1.835 11.17 11.17 0 0 0 3.812.66 9.594 9.594 0 0 0 4.095-.895 9.257 9.257 0 0 0 3.294-2.541l-3.765-2.777a7.146 7.146 0 0 1-1.6 1.553c-.612.377-1.365.611-2.26.611a4.14 4.14 0 0 1-2.681-.94c-.706-.613-1.177-1.46-1.412-2.447h12.517V67.13c0-1.412-.188-2.636-.564-3.765-.376-1.177-.941-2.165-1.647-2.965a6.836 6.836 0 0 0-2.588-1.836 8.875 8.875 0 0 0-3.39-.658zm20.47 0c-.658 0-1.27.094-1.787.236-.565.188-1.036.375-1.46.658a7.266 7.266 0 0 0-1.128.894c-.33.33-.565.659-.706.988h-.142v-2.4h-5.176V75.79h5.411v-8.66c0-.564.047-1.129.094-1.646.094-.518.236-.989.47-1.412.236-.424.566-.753.99-.988.423-.236.988-.377 1.694-.377s1.223.141 1.6.424c.376.282.658.659.846 1.13.189.47.283.94.33 1.505.047.517.047 1.082.047 1.553v8.564H92v-9.694c0-1.176-.094-2.258-.235-3.294-.236-.988-.518-1.882-.988-2.635-.47-.753-1.13-1.318-1.977-1.742-.847-.423-1.93-.61-3.294-.61zm34.307 0c-1.365 0-2.636.235-3.812.658a9.361 9.361 0 0 0-3.106 1.836c-.894.8-1.553 1.789-2.07 2.918-.518 1.13-.753 2.4-.753 3.765 0 1.411.235 2.635.753 3.764.517 1.13 1.176 2.118 2.07 2.918.894.8 1.93 1.412 3.106 1.835 1.176.424 2.447.66 3.812.66a9.59 9.59 0 0 0 4.093-.895 9.257 9.257 0 0 0 3.294-2.541l-3.764-2.777a7.142 7.142 0 0 1-1.6 1.553c-.611.377-1.365.611-2.259.611a4.14 4.14 0 0 1-2.682-.94c-.706-.613-1.176-1.46-1.411-2.447H128V66.8c-.047-1.271-.236-2.447-.565-3.482-.377-1.13-.941-2.118-1.647-2.918a6.836 6.836 0 0 0-2.589-1.836 8.87 8.87 0 0 0-3.387-.658zm-55.295 3.906c.989 0 1.789.329 2.4.988.612.658.942 1.411.895 2.353h-7.153v-.047c.047-.424.14-.847.33-1.224.187-.376.422-.752.752-1.035.33-.282.706-.565 1.176-.753.47-.188.989-.282 1.6-.282zm54.777 0c.988 0 1.788.329 2.4.988.612.658.941 1.411.894 2.353h-7.153v-.047c.047-.424.141-.847.33-1.224.188-.376.423-.752.752-1.035.33-.282.706-.565 1.177-.753.47-.188.988-.282 1.6-.282zm-106.26.847c-1.364 0-2.54.423-3.528 1.223-.989.8-1.647 1.883-2.023 3.2-.377 1.318-.33 2.4.235 3.2.47.847 1.412 1.224 2.823 1.224 1.365 0 2.542-.424 3.53-1.224s1.693-1.882 2.07-3.2c.376-1.317.282-2.4-.236-3.2-.517-.8-1.505-1.223-2.87-1.223z"/></svg>

```

Reply:
Seems like the original icon actually has no color, so this change is not needed